common/versioning/version.properties

```diff
@@ -0,0 +1,3 @@
+#Thu Aug 02 12:03:16 PDT 2018
+versionName=0.0.1
```

Comment:
I suspect the 'correct' thing to do here would be to start this value at `0.0.2` and rev to `0.0.3` when `release/0.0.3` merges to `dev`